bindings/gumjs/gumcmodule.c

```diff
@@ -63,21 +95,50 @@ static const gchar * gum_undecorate_name (const gchar * name);
 extern void * __va_arg (void * ap, int arg_type, int size, int align);
 #endif
 
+#define GUM_TYPE_TCC_CMODULE (gum_tcc_cmodule_get_type ())
+G_DECLARE_FINAL_TYPE (GumTccCModule, gum_tcc_cmodule, GUM, TCC_CMODULE,
+    GumCModule)
+G_DEFINE_TYPE (GumTccCModule, gum_tcc_cmodule, G_TYPE_OBJECT)
```

Comment:
```suggestion
G_DEFINE_TYPE (GumTccCModule, gum_tcc_cmodule, GUM_TYPE_CMODULE)
```

So the parent class is `Gum.CModule` instead of `GLib.Object`.